### PR TITLE
Moved sampleData file into a separate folder and changed all referenc…

### PR DESCRIPTION
**Pull request by jrauschenberg**
#### What's this PR do?
Moves the sampleData file (used for testing) into its own "sample_data" folder and changes all references to sampleData to reflect that move.
#### Where should the reviewer start?
sample_data/sampleData.js
#### How should this be manually tested?
Check all references to sampleData to make sure they reference the new location.
#### What are the relevant tickets?
[INT-166](https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-166)
#### Questions:

…es to sampleData file to match.

### CODE REVIEW

File: test/deleteShiftsAndRedirect.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var nock = require('nock'),
     WhenIWork = require('wheniwork-unofficial');
 
 var assert = require('assert')
-  , sampleData = require('./sampleData')
+  , sampleData = require(CONFIG.root_dir + './sample_data/sampleData')
```
Comment (mbogor) [inline test/deleteShiftsAndRedirect.js]: why './sample_data'? i get errors when i run this as is. it works if i remove the period before '/sample_data'; i get the same error for all other ones that include a period.

Comment (jrauschenberg) [inline test/deleteShiftsAndRedirect.js]: yeah, you are right. Curses! will fix it.

File: test/helpers/base.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-var sampleData = require('../sampleData');
+var sampleData = require(CONFIG.root_dir + './sample_data/sampleData');
```
Comment (mbogor) [inline test/helpers/base.js]: CONFIG isn't yet defined at this point and i get errors. basically, index.js requires the config file but if we're in the test environment, the config file requires this file (helpers/base.js) which in turn requires a CONFIG variable that wasn't yet run.

File: test/notifyMoreShifts.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 var assert = require('assert');
 var notifyMoreShifts = require('../jobs/scheduling/NotifyMoreShifts.js');
-var sampleData = require(CONFIG.root_dir + '/test/sampleData');
+var sampleData = require(CONFIG.root_dir + './sample_data/sampleData');
```
Comment (mbogor) [inline test/notifyMoreShifts.js]: i also get fun little error here like 'Cannot read property 'filter' of undefined' on line 85

**Comment by apackin** [comment]
The notifyMoreShifts test on master are still for the old version of notifyMoreShifts.
PR with updated test https://github.com/CrisisTextLine/Elmer/pull/129 .
This looks good to go 👍

**Comment by tongxiang** [comment]
When I run `npm test` on this branch, all tests are breaking. @jrauschenberg 

```
~/ctl/Elmer ((65ab016...)) $ npm test

> application-name@0.0.1 test /Users/tongxiang/ctl/Elmer
> NODE_ENV=test ./node_modules/.bin/mocha --reporter spec 'test/index.js'

index file running
/Users/tongxiang/ctl/Elmer/jobs/scheduling/NotifyMoreShifts.js:85
  return users.filter(function(user) {
              ^

TypeError: Cannot read property 'filter' of undefined
    at removeOlderUsers (/Users/tongxiang/ctl/Elmer/jobs/scheduling/NotifyMoreShifts.js:85:15)
    at /Users/tongxiang/ctl/Elmer/jobs/scheduling/NotifyMoreShifts.js:30:27
    at Object.WhenIWork.get (/Users/tongxiang/ctl/Elmer/test/helpers/base.js:17:7)
    at notifyMoreShifts (/Users/tongxiang/ctl/Elmer/jobs/scheduling/NotifyMoreShifts.js:25:14)
    at Object.<anonymous> (/Users/tongxiang/ctl/Elmer/jobs/scheduling/NotifyMoreShifts.js:16:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/tongxiang/ctl/Elmer/test/notifyMoreShifts.js:2:24)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/tongxiang/ctl/Elmer/test/index.js:19:24)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at /Users/tongxiang/ctl/Elmer/node_modules/mocha/lib/mocha.js:219:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/tongxiang/ctl/Elmer/node_modules/mocha/lib/mocha.js:216:14)
    at Mocha.run (/Users/tongxiang/ctl/Elmer/node_modules/mocha/lib/mocha.js:468:10)
    at Object.<anonymous> (/Users/tongxiang/ctl/Elmer/node_modules/mocha/bin/_mocha:403:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (node.js:348:7)
    at startup (node.js:140:9)
    at node.js:463:3
npm ERR! Test failed.  See above for more details.
```

**Comment by apackin** [comment]
@tongxiang : Please see my comment above

**Pull request merged**